### PR TITLE
Translation of group module

### PR DESCRIPTION
**Pull request by AyeRlS**
The group module has been marked for translation,

The build is currently failing because the elements have been marked for translation but there is not translation available for them

### CODE REVIEW

**Review by smadbe — CHANGES_REQUESTED** [review]

File: src/app/modules/group/components/group-invite-users/group-invite-users.component.ts [hunk 1]
```diff
@@ -86,29 +86,29 @@ export class GroupInviteUsersComponent implements OnInit, OnDestroy {
     if (successInvites.length > 0)
       this.messages.push({
         type: 'success',
-        summary: `${successInvites.length} user(s) invited successfully: `,
-        detail: `${successInvites.join(', ')}`,
+        summary: $localize`${successInvites.length} user(s) invited successfully: `,
+        detail: $localize`${successInvites.join(', ')}`,
```
[the remark this answers is not in the document]
Comment (AyeRlS) [reply src/app/modules/group/components/group-invite-users/group-invite-users.component.ts]: Fixed

File: src/app/modules/group/components/group-invite-users/group-invite-users.component.html [hunk 2]
```diff
@@ -1,14 +1,14 @@
-<alg-section-paragraph icon="fa fa-user-plus" label="Invite users" [collapsible]="true">
-  <span class="content-title">Add users name list, separated by commas.</span>
+<alg-section-paragraph icon="fa fa-user-plus" i18n-label i18n-label label="Invite users" [collapsible]="true">
```
[the remark this answers is not in the document]
Comment (AyeRlS) [reply src/app/modules/group/components/group-invite-users/group-invite-users.component.html]: Fixed

File: src/app/modules/group/components/group-composition-filter/group-composition-filter.component.ts [hunk 3]
```diff
@@ -88,14 +88,14 @@ export class GroupCompositionFilterComponent implements OnInit{
 
   onTypeFilterChanged(index: number): void {
     const typeFilters = this.value.directChildren ? this.directChildrenTypeFilters : this.allDescendantsTypeFilters;
-    if (index < 0 || index >= typeFilters.length) throw Error('invalid value for type filter');
+    if (index < 0 || index >= typeFilters.length) throw Error($localize`invalid value for type filter`);
```
[the remark this answers is not in the document]
Comment (AyeRlS) [reply src/app/modules/group/components/group-composition-filter/group-composition-filter.component.ts]: Fixed

File: src/app/modules/group/components/group-join-by-code/group-join-by-code.component.html [hunk 4]
```diff
@@ -1,16 +1,16 @@
...
     <p-progressSpinner *ngIf="processing"></p-progressSpinner>
 
     <div class="code-state" *ngIf="!processing && groupExt">
-      <span class="label" *ngIf="groupExt.hasCodeNotSet">There is currently no code set.</span>
-      <span class="label" *ngIf="groupExt.hasCodeUnused">This code has not been used yet (or usage resetted).</span>
+      <span class="label" *ngIf="groupExt.hasCodeNotSet" i18n>There is currently no code set.</span>
+      <span class="label" *ngIf="groupExt.hasCodeUnused" i18n>This code has not been used yet (or usage resetted).</span>
       <span class="label green" *ngIf="groupExt.hasCodeInUse">
         This code has been activated {{ groupExt.durationSinceFirstCodeUse! | toMin | number:'1.0-0' }} minutes ago,
         it will be disabled in {{ groupExt.durationBeforeCodeExpiration! | toMin | number:'1.0-0' }} minutes.
```
Comment (smadbe) [inline src/app/modules/group/components/group-join-by-code/group-join-by-code.component.html:16]: And this one? And the 2 lines below?

File: src/app/modules/group/components/joined-group-list/joined-group-list.component.html [hunk 5]
```diff
@@ -13,8 +13,8 @@
   >
   <ng-template pTemplate="header" let-columns>
     <tr *ngIf="data.length > 0">
-        <th>Name</th>
-        <th>Type</th>
+        <th i18n>Name</th>
+        <th i18n>Type</th>
         <th pSortableColumn="member_since">Joined On <p-sortIcon field="member_since"></p-sortIcon></th>
```
Comment (smadbe) [inline src/app/modules/group/components/joined-group-list/joined-group-list.component.html:18]: And this one?

**Review by smadbe — CHANGES_REQUESTED** [review]
Looks like some strings are missing, check everything carefully.

File: src/app/modules/group/components/joined-group-list/joined-group-list.component.html (hunk 5 above)
Comment (AyeRlS) [reply src/app/modules/group/components/joined-group-list/joined-group-list.component.html]: Fixed

File: src/app/modules/group/components/group-join-by-code/group-join-by-code.component.html (hunk 4 above)
Comment (AyeRlS) [reply src/app/modules/group/components/group-join-by-code/group-join-by-code.component.html:16]: Fixed

**Review by smadbe — COMMENTED** [review]
Ok, I'll do the translations.

**Review by smadbe — APPROVED** [review]
You can merge. Resolve **carefully** the conflicts first for the xlf files.

File: src/locale/messages.fr.xlf [hunk 6]
```diff
@@ -55,7 +55,14 @@
          <context context-type="sourcefile">src/app/modules/group/components/pending-request/pending-request.component.html</context>
          <context context-type="linenumber">73</context>
        </context-group>
-     </trans-unit><trans-unit id="6fc5e65900ae1415d3170d5d2842f0dcae1b6645" datatype="html">
+     </trans-unit><trans-unit id="c27357089524343d4da96cb643ac6efc6130a581" datatype="html">
+      <source> Select all </source>
+      <target state="translated"> Tous sélectionner </target>
```
Comment (smadbe) [inline src/locale/messages.fr.xlf:60]: tout

**Review by smadbe — CHANGES_REQUESTED** [review]

**Review by smadbe — APPROVED** [review]

**Pull request merged**